Xamarin.Essentials/SecureStorage/SecureStorage.android.cs

```diff
@@ -26,7 +26,15 @@ static Task<string> PlatformGetAsync(string key)
             {
                 var encData = Convert.FromBase64String(encStr);
                 var ks = new AndroidKeyStore(context, Alias, AlwaysUseAsymmetricKeyStorage);
-                decryptedData = ks.Decrypt(encData);
+                try
+                {
+                    decryptedData = ks.Decrypt(encData);
+                }
+                catch (AEADBadTagException)
+                {
+                    System.Diagnostics.Debug.WriteLine($"Unable to decrpt key {key} likely due to uninstall, removing old key.");
```

Comment:
This is OK here, but since we compile in release mode, I am thinking that if we want other devs to see this, we may have to use the android log. This probably goes for the other platforms - we should use the platform logging if we need the developer to see these errors and things.

Reply:
Yeah, my thought was that we probably don't want to leak the key's name in release mode.